autovalet_husky/src/autovalet_husky/find_parking_spot.py

```diff
@@ -107,7 +112,7 @@ def tag_pose_RANSAC(self):
             for j in range(len(quatArray)):
                 diff = np.abs(1 - np.dot(quatArray[i], quatArray[j]) ** 2)
                 vote = vote + 1 if diff < 0.005 else vote
-                if i == 0:
+                if i == 0 and self.debug:
                     print(diff)
```

Comment:
sorry, this msg shouldn't even be there. I forgot to remove it after debugging, can you remove those two lines for me?

Reply:
removed